src/openforms/formio/tasks.py

```diff
@@ -0,0 +1,88 @@
+import traceback
+
+import structlog
+from celery import group
+
+from openforms.celery import app
+from openforms.config.models import GlobalConfiguration
+from openforms.logging import logevent
+from openforms.submissions.constants import ComponentPreRegistrationStatuses
+from openforms.submissions.models import Submission
+
+from .registry import register as formio_registry
+from .typing.base import Component
+
+logger = structlog.stdlib.get_logger(__name__)
+
+
+@app.task
+def pre_registration_component_task(component: Component, submission_id: int) -> None:
+    component_key = component["key"]
+    log = logger.bind(
+        action="formio.component_pre_registration",
+        submission_id=submission_id,
+        component=component_key,
```

Comment:
I think the `component type` is more interesting to log than the key, but doing both should be okay I think.